FirebaseAnalytics/README.md

```diff
@@ -1,2 +1,111 @@
-This directory open sources select files from the Firebase Analytics SDK. Note
-that there is no open source infrastructure to build or package them.
+# Firebase Analytics Swift SDK
```

Comment:
Change title

Reply:
Done in https://github.com/firebase/firebase-ios-sdk/pull/13056/commits/8422ef6e081eae7a69c28d0528e4c4d6ea6cfdee